src/esp/assets/ResourceManager.h

```diff
@@ -323,6 +333,55 @@ class ResourceManager {
   std::unique_ptr<MeshData> createJoinedCollisionMesh(
       const std::string& filename);
 
+  /**
+   * @brief Get primitive mesh data (points and triangle arrays) from a
+   * MeshMetaData filename.
+   *
+   * @param filename The MeshMetaData filename to receive primitive attributes
+   * from.
+   * @param points The vector of points to be filled with the mesh vertices.
+   * @param triangles The vector of indices to be filled with the mesh index
+   * buffer.
+   */
+  void getPrimitiveMeshData(const std::string& filename,
```

Comment:
This is a minor thing: since this doesn't "get"/return anything, can we change the name to something like buildPrimitiveMeshData? Also, mark `points` and `triangles` as [in] variables in docs, please :).

Reply:
Good point. As far as the docs go, what is the difference between @param[in] and @param[out]?

Reply:
Oops.  that should have been "[out]", my mistake.